dcpy/connectors/edm/publishing.py

```diff
@@ -30,6 +30,22 @@
 BUCKET = PUBLISHING_BUCKET
 
 BASE_DO_URL = f"https://cloud.digitalocean.com/spaces/{BUCKET}"
+LOGGING_DB = "edm-qaqc"
+LOGGING_SCHEMA = "product_data"
+LOGGING_TABLE_NAME = "event_logging"
+PRODUCTS_TO_LOG = [
```

Comment:
this seems like a good idea to have a "whitelist" of products to log, rather than a list of products to ignore. should prevent surprise over-logging

Reply:
I do think the general move has been towards making dcpy _more_ agnostic about the products, so I'm hesitant about this living here. I'd prefer that it were defined in a configuration, similar to how @fvankrieken configured buckets in that recent PR.

Reply:
@alexrichey, do you think `LOGGING_DB`, `LOGGING_SCHEMA`, `LOGGING_TABLE_NAME`, and `IGNORED_LOGGING_BUILDS` variables should be moved to the config file as well?

Reply:
Yeah, I think we should move those.